extensions/azurecore/src/azureResource/azure-resource.d.ts

```diff
@@ -3,8 +3,9 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { TreeDataProvider, TreeItem } from 'vscode';
+import { TreeDataProvider } from 'vscode';
 import { DataProvider, Account } from 'sqlops';
+import { TreeItem } from 'sqlops';
```

Comment:
was this removed from `vscode`?  or we're adding a new implementation in `sqlops`/  If we're adding a new implementation, have we considered renaming so they can be easily distinguished when reading code?

Reply:
Its an extended version of vscode TreeItem. I was trying to keep the differences between vscodes and ads tree implement minimal